ring-core/src/ring/middleware/file.clj

```diff
@@ -40,15 +40,29 @@
   Accepts the following options:
 
   :index-files?    - look for index.* files in directories, defaults to true
-  :allow-symlinks? - serve files through symbolic links, defaults to false"
+  :allow-symlinks? - serve files through symbolic links, defaults to false
+  :prefer-handler? - give preference to handler to process the file. 
```

Comment:
Can you change this to:

    :prefer-handler? - prioritize handler response over files, defaults to false